internal/app/go-filecoin/node/testing.go

```diff
@@ -202,23 +200,27 @@ var PeerKeys = []crypto.PrivKey{
 	mustGenKey(102),
 }
 
-// TestGenCfg is a genesis configuration used for tests.
-var TestGenCfg = &gengen.GenesisCfg{
-	ProofsMode: types.TestProofsMode,
-	Keys:       2,
-	Miners: []*gengen.CreateStorageMinerConfig{
-		{
-			Owner:               0,
-			NumCommittedSectors: 100,
-			PeerID:              mustPeerID(PeerKeys[0]).Pretty(),
-			SectorSize:          constants.DevSectorSize,
+// MakeTestGenCfg returns a genesis configuration used for tests.
+func MakeTestGenCfg(t *testing.T) *gengen.GenesisCfg {
+	commCfgs, err := gengen.MakeNCommitCfgs(100)
```

Comment:
Could we describe the intended configuration in english too? E.g. "One miner and two accounts (the first is the miner's owner/worker) with 100 sectors. the accounts both have 10000 FIL"

Take the 100 as a parameter if possible. Parameters for more things would help decouple the tests from each other, otherwise various tests will lock in parts of this setup that are then make other tests difficult. (low priority)

Reply:
Sounds great